src/util/logs.rs

```diff
@@ -0,0 +1,64 @@
+use crate::subscriptions;
+use colored::Colorize;
+
+pub fn format_attr_log(mut log: subscriptions::deployment_logs::LogFields) {
+    if !log.attributes.is_empty() {
+        let mut timestamp: Option<String> = None;
+        let mut level: Option<String> = None;
+        let message = log.message;
+        let mut others = Vec::new();
+        // for some reason, not all have "" around the value
+        for attr in &mut log.attributes {
+            if !attr.value.starts_with('"') {
+                attr.value.insert(0, '"');
+            };
+            if !attr.value.ends_with('"') {
+                attr.value.push('"');
+            }
+        }
+        // get attributes using a match
+        for attr in &log.attributes {
+            match attr.key.to_lowercase().as_str() {
+                "timestamp" | "ts" | "time" => {
+                    timestamp = Some(attr.value.clone().replace('"', ""))
+                }
```

Comment:
I would prefer to use `log.timestamp` instead of the one from the attributes, to be consistent with dashboard behavior

Reply:
if a custom timestamp is provided, should i just skip over it?